### PR TITLE
Hyperledger Aries 2024 Q4 Quarterly Report

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Review by petermetz — APPROVED** [review]

File: tac/project-updates/2024/2024-Q4-Hyperledger-Aries.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+---
+
+## Project Health
+
+Work on [Hyperledger Aries] related projects continues at a healthy pace, albeit
+with some of that work now outside of the Aries itself. [ACA-Py] (formerly Aries
+Cloud Agent Python) achieved two significant milestones this quarter. Release
+1.0.0 was published, and the project moved to the [OpenWallet Foundation].
+Within [Hyperledger Aries], progress was made on all of the frameworks -- [Aries VCX],
+[Aries Framework Swift] and [Aries Framework Kotlin]. Other [Open Wallet
```
Comment (tkuhrt) [inline tac/project-updates/2024/2024-Q4-Hyperledger-Aries.md:15]:
```suggestion
[Aries Framework Swift] and [Aries Framework Kotlin]. Other [OpenWallet
```

File: tac/project-updates/2024/2024-Q4-Hyperledger-Aries.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+layout: default
+title: 2024 Q4 Hyperledger Aries
+parent: 2024
+grand_parent: Project Updates
+---
+
+## Project Health
+
+Work on [Hyperledger Aries] related projects continues at a healthy pace, albeit
+with some of that work now outside of the Aries itself. [ACA-Py] (formerly Aries
```
[the remark this answers is not in the document]
Comment (swcurran) [reply tac/project-updates/2024/2024-Q4-Hyperledger-Aries.md:11]: I think the correct way is "Aries" (remove the "the"). The project is still "Hyperledger Aries".

File: tac/project-updates/2024/2024-Q4-Hyperledger-Aries.md (hunk 1 above)
Comment (swcurran) [reply tac/project-updates/2024/2024-Q4-Hyperledger-Aries.md:15]: The title of the Foundation on the website has no space -- see: https://openwallet.foundation/

File: tac/project-updates/2024/2024-Q4-Hyperledger-Aries.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+[Hyperledger Aries]: https://www.lfdecentralizedtrust.org/projects/aries
+[Credo-TS]: https://github.com/openwallet-foundation/credo-ts
+[ACA-Py Repository]: https://github.com/openwallet-foundation/acapy
+[Aries VCX]: https://github.com/hyperledger/aries-vcx
+[OpenWallet Foundation]: https://openwallet.foundation/
+[Aries Framework Swift]: https://github.com/hyperledger/aries-framework-swift
+[Aries Framework Kotlin]: https://github.com/hyperledger/aries-framework-kotlin
+[Askar]: https://github.com/openwallet-foundation/askar
+
+## Questions/Issues for the TOC
```
[the remark this answers is not in the document]
Comment (swcurran) [reply tac/project-updates/2024/2024-Q4-Hyperledger-Aries.md:41]: Fixed

File: tac/project-updates/2024/2024-Q4-Hyperledger-Aries.md (hunk 1 above)
Comment (tkuhrt) [reply tac/project-updates/2024/2024-Q4-Hyperledger-Aries.md:15]: Agree. Removing the space in my suggestion.

Comment (swcurran) [reply tac/project-updates/2024/2024-Q4-Hyperledger-Aries.md]: Fixed

**Review by tkuhrt — APPROVED** [review]

**Review by VRamakrishna — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Pull request merged**